### PR TITLE
Apply (new) RefMan conventions

### PR DESCRIPTION
**Pull request by FabienLelaquais**

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]
🥇

**Review by dinhlongviolin1 — APPROVED** [review]
Dont really understand the docs syntax so I am just gonna approve it :D

File: taipy/gui/gui.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def __init__(
                 The method `(Gui.)add_pages()^` is called if _pages_ is not None, and
                 you can find details on the possible values of this argument in the
                 documentation for this method.
```
Comment (florian-vuillemot) [inline taipy/gui/gui.py]:
```suggestion
                The method `(Gui.)add_pages()^` is called if _pages_ is not None.
                You can find details on the possible values of this argument in the
                documentation for this method.
```

Comment (FabienLelaquais) [reply taipy/gui/gui.py]: Absolutely. Thx.

**Pull request merged**